### PR TITLE
Update deployment with one line deploy guide

### PR DESCRIPTION
**Pull request by fozy81**
Updated deployment guide using hoodie blog post as basis: http://hood.ie/blog/single-line-deployment.html

### CODE REVIEW

File: docs/guides/deployment.rst [hunk 1]
```diff
@@ -1,27 +1,64 @@
...
-Unfortunately, now is a read-only file system, so you must either run
-your app in memory or set an external CouchDB URL.
+- hoodiehq/hoodie-app-tracker, is the GitHub repository slug.
 
-Add this script to your package.json and you are good to go:
+- --npm tells now to deploy using npm, as there is also Dockerfile in the repository.
+
+- -e NODE_ENV=production sets the NODE_ENV environment variable to production, which makes the deployment faster as no devDependencies will be installed. 
+
+- -e hoodie_inMemory=true makes the Hoodie app run in-memory mode, meaning that no data is persisted and no files are written. This is important because now is a read-only file system. That means that all user accounts and data will be lost on the next deployment, but it is great for for a quick test or demo of your application.
```
Comment (gr2m) [inline docs/guides/deployment.rst]: What do you think about doing inline code highlights for `hoodie/hoodie-app-tracker`, `--npm`, `-e NODE_ENV=production` and `-e hoodie_inMemory=true`? You can do it with 

```
:code:`highlight here`
```

File: docs/guides/deployment.rst [hunk 2]
```diff
@@ -1,27 +1,64 @@
...
+.. _Now: https://zeit.co/now
+Store Data With Cloudant
+~~~~~~~~~~~~~~~~~~~~~~~~
+`Cloudant`_ is a DBaaS (database-as-a-service). It provides most of CouchDB’s APIs and can be used as Hoodie’s database backend. Signing up for a free account only takes a moment. After sign up, you need to slightly adjust the now deployment command above.
+::
+
+$ now hoodiehq/hoodie-app-tracker -e NODE_ENV=production -e hoodie_inMemory=true -e hoodie_dbUrl=https://username:password@username.cloudant.com/,
+
+The hoodie_inMemory environment variable makes sure that Hoodie does not try to write any files like the bundled /hoodie/client.js library. The hoodie_dbUrl environment variable sets the address and credentials to your CouchDB. Replace username and password to whatever you signed up with.
+..  _Cloudant: https://cloudant.com/
```
Comment (gr2m) [inline docs/guides/deployment.rst]: two spaces between `..` and `_Cloudant`

File: docs/guides/deployment.rst [hunk 3]
```diff
@@ -1,27 +1,64 @@
...
-deployment set at `hoodie-app-tracker/deployment.md`_.
+After you’ve built your Hoodie app you probably want to put it online. You can choose to deploy your app as read-only or deploy the backend couchdb database as well. This `video`_ and the text below descibes how to deploy your app using one line of code. Alternatively, you can deploy your app using Docker, please refer to the Docker section. 
 
-.. _Hoodie Tracker App: https://github.com/hoodiehq/hoodie-app-tracker
-.. _hoodie-app-tracker/deployment.md: https://github.com/hoodiehq/hoodie-app-tracker/blob/maste/deployment.md
+.. _video: https://youtu.be/29Uclxq_1Vw
+Deploying to Now
+~~~~~~~~~~~~~~~~
+.. _command line tool: https://github.com/zeit/now-cli
+`Now`_ allows you to deploy a Node application with its `command line tool`_. It’s 100% free for Open Source projects. You can deploy an app from your computer or right from a GitHub repository. For example, to deploy our Hoodie Tracker demo app all you have to do is to run this command:
```
Comment (gr2m) [inline docs/guides/deployment.rst]: Could you link `Hoodie Tracker demo` to https://github.com/hoodiehq/hoodie-app-tracker ?

File: docs/guides/deployment.rst [hunk 4]
```diff
@@ -1,27 +1,64 @@
...
-.. _now: https://zeit.co/now
+.. _Now: https://zeit.co/now
+Store Data With Cloudant
+~~~~~~~~~~~~~~~~~~~~~~~~
+`Cloudant`_ is a DBaaS (database-as-a-service). It provides most of CouchDB’s APIs and can be used as Hoodie’s database backend. Signing up for a free account only takes a moment. After sign up, you need to slightly adjust the now deployment command above.
+::
+
+$ now hoodiehq/hoodie-app-tracker -e NODE_ENV=production -e hoodie_inMemory=true -e hoodie_dbUrl=https://username:password@username.cloudant.com/,
+
+The hoodie_inMemory environment variable makes sure that Hoodie does not try to write any files like the bundled /hoodie/client.js library. The hoodie_dbUrl environment variable sets the address and credentials to your CouchDB. Replace username and password to whatever you signed up with.
```
Comment (gr2m) [inline docs/guides/deployment.rst]: can you highlight `hoodie_inMemory` and `hoodie_dbUrl`?

**Review by gr2m — CHANGES_REQUESTED** [review]
This is great, thanks so much! I’ve left a few comments, could you please have a look and let me know if it all makes sense? They are merely suggestions to further improve your additions :)

**Comment by fozy81** [comment]
@gr2m Thanks for your feedback, I've updated with your suggestions/fixes. I'm just getting use to the .rst markup - little bit of a learning curve.

**Comment by gr2m** [comment]
Thanks! And agree for .rst, same here! We decided for it because we try to make it as simple to contribute / maintain the docs as possible to folks doing documentation a lot, and they seem to like .rst a lot, and it’s also the default for readthedocs.org

**Review by gr2m — APPROVED** [review]

**Comment by gr2m** [comment]
it’s already up: http://docs.hood.ie/en/latest/guides/deployment.html thanks a ton :)

**Comment by fozy81** [comment]
wow, that's great. Happy to help out on similar things in future.

**Pull request merged**